mavros_extras/src/plugins/distance_sensor.cpp

```diff
@@ -183,6 +220,10 @@ class DistanceSensorPlugin : public MavRosPlugin {
 void DistanceSensorItem::range_cb(const sensor_msgs::Range::ConstPtr &msg)
 {
 	uint8_t type = 0;
+	uint8_t covariance_ = 0;
+
+	if (cov_is_def) covariance_ = covariance;
+	else covariance_ = (uint8_t) DistanceSensorItem::calculate_variance(msg->range / 1E-2);
```

Comment:
Maybe better convert to centimetres result? Try to avoid C-style casting. Also i think there no need `DistanceSensorItem::`.